tests/main/snapshot-exclusions-dynamic/task.yaml

```diff
@@ -0,0 +1,107 @@
+summary: Check the basic dynamic snapshot data exclusion functionality
+
+environment:
+    # tar invoked when creating snapshots triggers OOM
+    SNAPD_NO_MEMORY_LIMIT: 1
+    USER_DATA_DIR: /root/snap/test-snap
+    SYSTEM_DATA_DIR: /var/snap/test-snap
+
+prepare: | 
+    "$TESTSTOOLS"/snaps-state install-local test-snap
+    snap install --devmode --edge test-snapd-curl
+    snap install --edge jq
+    snap alias jq edge-jq
```

Comment:
This line is not needed

Reply:
Hi Sergio. I had a problem on 14.04 with an older version of `jq` that have an issue with formatting of filter notation. I then tried installing edge which works, but it clashes with the already installed jq. This is why I created an alias. What am I missing here?

Reply:
in this case just skip the test in 14.04 because it is not longer supported for this feature.

We already disabled many tests in 14.04 to avoid having more complex test.